Safety/AttitudeManager/Src/OutputMixing.cpp

```diff
@@ -82,11 +82,11 @@ static OutputMixingErrorCodes checkInputValidity(PID_Output_t *PidOutput)
 {
 	OutputMixingErrorCodes errorCode;
 
-	if ( (PidOutput->frontLeft < -100.0f) || (PidOutput->pitchPercent < -100.0f) || (PidOutput->rudderPercent < -100.0f) || (PidOutput->throttlePercent < 0.0f) )
+	if ( (PidOutput->frontLeftMotorPercent < -100.0f) || (PidOutput->frontRightMotorPercent < -100.0f) || (PidOutput->backLeftMotorPercent < -100.0f) || (PidOutput->backRightMotorPercent < 0.0f) )
```

Comment:
Can we break these onto a few lines so that it's not so hard to read?

Reply:
Will do